lib/cli/analytics.js

```diff
@@ -16,10 +16,21 @@ phonegap analytics reset|on|off
 */
 
 var Insight = require('insight');
+var fs = require('fs');
+var path = require('path');
+
+// Google Analytics tracking code
+var trackingCode = 'UA-94271-37'; // this is the default production tracking code
+
+var rootPath = require.resolve('../../package.json');
+var gitPath = path.join(path.dirname(rootPath),'.git');
+if(fs.existsSync(gitPath)) {
+    // set it to the dev tracking code
+    trackingCode = 'UA-94271-38';
+}
+
 var insight = new Insight({
-    // Google Analytics tracking code
-    // for release, use 'UA-94271-37'
-    "trackingCode": 'UA-94271-38',
+    "trackingCode": trackingCode,
     "pkg": require('../../package.json')
```

Comment:
Object literals should have double quotes on property names. This comes from the JSON spec.
Also, that is not a part of this pr.